src/container/srv_container.c

```diff
@@ -603,9 +621,10 @@ cont_create(struct rdb_tx *tx, struct ds_pool_hdl *pool_hdl,
 				in->cci_op.ci_uuid));
 		D_GOTO(out_kvs, rc = -DER_NOMEM);
 	}
-	rc = cont_prop_default_copy(prop_dup, in->cci_prop);
+	pm_ver = pool_map_get_version(pool_hdl->sph_pool->sp_map);
+	rc = cont_create_prop_prepare(prop_dup, in->cci_prop, pm_ver);
```

Comment:
How do we make sure any write to this container will be done with a pool map version >= pm_ver?

- Looks like we are not filling cont_op_out.co_map_version.
- dc_cont_glob does not include a pool map version field.

If there's a write to this container done with a lower pool map version, what may happen?

Reply:
I think this is a good question and valid concern. If there is a write based on older pm_ver, it possibly cause that data lost cannot be detected, because here we store the pm_ver when cont_create and later only detect failures after this pm_ver.

My current thought is - read out the pm_ver that created the container when cont_open (and yes put in cont l2g/g2l), and client's cont_open's completion check client-side cached pm_ver, fresh pool map if client pm ver < cont_create's pm ver. Also for cont g2l, also possibly need to fresh pool map if need.
What do you think?  
And I suppose it can be done by a following patch, as this patch is already quite big.

Reply:
I see. Yes, makes sense to me.